Sources/Playbook/Components/Label Pill/PBLabelPill.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  PBLabelPill.swift
+//
+
+import SwiftUI
+
+public struct PBLabelPill: View {
+  let variant: PBPill.Variant
+  let label: String?
+  let pillValue: String?
+  let offset: Double?
+  let horizontalPadding: CGFloat?
+  public init(
+    variant: PBPill.Variant = .neutral,
+    label: String? = nil,
+    pillValue: String? = "76",
+    offset: Double? = -5.5,
+    horizontalPadding: CGFloat? = Spacing.xSmall
+  ) {
+    self.variant = variant
+    self.label = label
+    self.pillValue = pillValue
+    self.offset = offset
+    self.horizontalPadding = horizontalPadding
+  }
+  
+  public var body: some View {
+    pillLabelView
+  }
+}
+
+public extension PBLabelPill {
+  var pillLabelView: some View {
+    HStack(spacing: horizontalPadding) {
+      labelView
+      pillView
+    }
+  }
+  @ViewBuilder
+  var labelView: some View {
+    if let label = label {
+      if let offset = offset {
```

Comment:
is this offset always necessary? If we have a label and a nil offset, the label will be hidden.

Reply:
If we dont have it, it won't be lined up correctly but I can fix it so we have a label if the offset is nil(maybe)

Reply:
why don't remove the offset and use the space on the HStack?